hack/ci/e2e-fedora.sh

```diff
@@ -16,9 +16,11 @@
 set -euo pipefail
 
 export E2E_CLUSTER_TYPE=vanilla
-export E2E_TEST_SELINUX=true
+# TODO: re-enable when SELinux tests are fixed
+export E2E_TEST_SELINUX=false
```

Comment:
@jhrozek It looks like that the raw-errorlogger policy status it not getting reconciled correctly during our e2e tests:

```
[vagrant@localhost vagrant]$ k get rawselinuxprofile -A
NAMESPACE   NAME              USAGE   STATE
default     raw-errorlogger
```

```
[vagrant@localhost vagrant]$ k logs spod-4lbs2 -c selinuxd | tail
{"level":"info","ts":1706615122.7534387,"logger":"file-watcher","caller":"daemon/daemon.go:93","msg":"Installing policy","file":"/etc/selinux.d/errorlogger_default.cil"}
{"level":"info","ts":1706615127.4619582,"logger":"policy-installer","caller":"daemon/daemon.go:131","msg":"The operation was successful","operation":"install - /etc/selinux.d/errorlogger_default.cil"}
{"level":"info","ts":1706615127.4627874,"logger":"policy-installer","caller":"daemon/daemon.go:131","msg":"The operation was successful","operation":"install - /etc/selinux.d/errorlogger_default.cil"}
{"level":"info","ts":1706615167.7674167,"logger":"file-watcher","caller":"daemon/daemon.go:90","msg":"Removing policy","file":"/etc/selinux.d/errorlogger_default.cil"}
{"level":"info","ts":1706615167.8494902,"caller":"semanage/semanage.go:89","msg":"Removing last errorlogger_default module (no other errorlogger_default module exists at another priority)."}
{"level":"info","ts":1706615172.4159262,"logger":"policy-installer","caller":"daemon/daemon.go:131","msg":"The operation was successful","operation":"remove - /etc/selinux.d/errorlogger_default.cil"}
{"level":"info","ts":1706615173.1398582,"logger":"file-watcher","caller":"daemon/daemon.go:93","msg":"Installing policy","file":"/etc/selinux.d/raw-errorlogger_default.cil"}
{"level":"info","ts":1706615173.1398923,"logger":"file-watcher","caller":"daemon/daemon.go:93","msg":"Installing policy","file":"/etc/selinux.d/raw-errorlogger_default.cil"}
{"level":"info","ts":1706615177.8355806,"logger":"policy-installer","caller":"daemon/daemon.go:131","msg":"The operation was successful","operation":"install - /etc/selinux.d/raw-errorlogger_default.cil"}
{"level":"info","ts":1706615177.8360062,"logger":"policy-installer","caller":"daemon/daemon.go:131","msg":"The operation was successful","operation":"install - /etc/selinux.d/raw-errorlogger_default.cil"}
```

It hangs and fails there on the `wait`:

```
2024-01-30T10:44:20.8987852Z I0130 10:44:20.877194   11818 suite_test.go:701] "The 'errorlogger' workload should be able to use SELinux policy"
2024-01-30T10:44:20.8988560Z I0130 10:44:20.877511   11818 suite_test.go:701] "creating policy"
2024-01-30T10:44:21.3026430Z rawselinuxprofile.security-profiles-operator.x-k8s.io/raw-errorlogger created
2024-01-30T10:50:21.5070585Z error: timed out waiting for the condition on rawselinuxprofiles/raw-errorlogger
2024-01-30T10:50:21.5271344Z     suite_test.go:588: 
2024-01-30T10:50:21.5272088Z         	Error Trace:	/vagrant/test/suite_test.go:588
2024-01-30T10:50:21.5272813Z         	            				/vagrant/test/suite_test.go:612
2024-01-30T10:50:21.5406174Z         	            				/vagrant/test/tc_selinux_base_usage_test.go:172
2024-01-30T10:50:21.5408701Z         	            				/vagrant/test/tc_selinux_base_usage_test.go:159
2024-01-30T10:50:21.5410889Z         	            				/vagrant/test/e2e_test.go:162
2024-01-30T10:50:21.5413079Z         	            				/vagrant/vendor/github.com/stretchr/testify/suite/suite.go:112
2024-01-30T10:50:21.5417728Z         	Error:      	Expected nil, but got: &errors.errorString{s:"command /usr/bin/kubectl wait --timeout 360s --for condition=ready rawselinuxprofile raw-errorlogger did not succeed: error: timed out waiting for the condition on rawselinuxprofiles/raw-errorlogger\n"}
2024-01-30T10:50:21.5422609Z         	Test:       	TestSuite/TestSecurityProfilesOperator/cluster-wide:_SELinux:_base_case_(install_policy,_run_pod_and_delete)_for_the_raw_CR
2024-01-30T10:50:21.8707125Z I0130 10:50:21.867988   11818 suite_test.go:701] "assert policy is installed"
```